docs/source/markdown/podman.1.md

```diff
@@ -1,56 +1,52 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for pods, containers and images
+podman - simple management tool for pods, containers and images
 
 ## SYNOPSIS
-**podman** [*options*] *command*
+**podman command** [*optional*] *value*
 
 ## DESCRIPTION
-Podman (Pod Manager) is a fully featured container engine that is a simple daemonless tool.
-Podman provides a Docker-CLI comparable command line that eases the transition from other
-container engines and allows the management of pods, containers and images.  Simply put: `alias docker=podman`.
-Most Podman commands can be run as a regular user, without requiring additional
-privileges.
+The podman command is used to control the Podman container engine using a CLI.  Most commands can be run as a regular user, without requiring additional privileges. Default settings for flags are defined in `containers.conf` which can be found in `/etc/containers`. Most settings for remote connections use the server’s `containers.conf`, except when documented in man pages.
```

Comment:
containers.conf is installed in /usr/share/containers/containers.conf not in /etc/containers.

Reply:
On my podman installation:
```
root@ae8790df7bd5:/usr/share/containers# ls
seccomp.json
```
```
root@ae8790df7bd5:/etc/containers# ls
containers.conf  policy.json  registries.conf
```
"Container engines read the /usr/share/containers/containers.conf and /etc/containers/containers.conf files if they exists. When running in rootless mode, they also read $HOME/.config/containers/containers.conf files."
From: https://github.com/containers/common/blob/master/docs/containers.conf.5.md
What would be the correct one?

Reply:
Well that is a packaging issue for your distribution, the way it is supposed to work is 

/usr/share/containers/containers.conf  - Comes from the distribution.
/etc/containers/containers.conf - Is modifications generated by the system admin
$HOME/.config/containers/containers.conf - Is for modifications by the user.

Reply:
Understood!